packages/js/plugins/uri-resolvers/ipfs-resolver/src/index.ts

```diff
@@ -46,62 +47,38 @@ export class IpfsResolverPlugin extends Module<NoConfig> {
       );
 
       if (manifestResult.ok) {
-        manifest = Buffer.from(manifestResult.value);
+        try {
+          manifest = Buffer.from(manifestResult.value);
+          return { uri: null, manifest: manifest };
+        } catch (e) {
+          break;
+        }
```

Comment:
Why do we have try...catch... here?

Reply:
Would conditionals work for this?

Reply:
The try catch was always there. I just moved it to be only around the only line that can throw. It is necessary because uri resolver extensions never throw; they return `null` instead. All of them work this way. 

There is an issue to address this: https://github.com/polywrap/toolchain/issues/663

And the specific try catch block here is temporary anyway, because there is another PR up that will replace the ipfs resolver plugin with a Wasm wrapper. I would actually love to get that one merged first since there will be merge conflicts. (Also, I fixed the reliability issue so we won't have problems with ipfs URIs not resolving).

Reply:
It's just `Buffer.from` and I feel like it maybe handled without try...catch by checking the value but it's alright if we are going to replace if with wrapper anyway

Reply:
Yeah, you're right. I removed the try-catch.